docs/node-groups.md

```diff
@@ -5,6 +5,43 @@ The framework uses dedicated sub modules for creating [AWS Managed Node Groups](
 The `aws-auth` ConfigMap handled by this module allow your nodes to join your cluster, and you also use this ConfigMap to add RBAC access to IAM users and roles.
 Each Node Group can have dedicated IAM role, Launch template and Security Group to improve the security.
 
+## Additional IAM Roles, Users and Accounts
+Access to EKS cluster using AWS IAM entities is enabled by the [AWS IAM Authenticator](https://docs.aws.amazon.com/eks/latest/userguide/install-aws-iam-authenticator.html) for Kubernetes, which runs on the Amazon EKS control plane. 
+The authenticator gets its configuration information from the `aws-auth` [ConfigMap](https://kubernetes.io/docs/concepts/configuration/configmap/)
+
+The below example demonstrates adding additional IAM Roles, IAM Users and Accounts using EKS Blueprints module
```

Comment:
Add a note on how this is different than teams.

Reply:
Added a paragraph for teams best practise and links